.github/workflows/android.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
     build:
+        if: github.user == 'botify'
```

Comment:
I think you want [OSBotify](https://github.com/OSBotify) here. Example: https://github.com/Expensify/Expensify.cash/actions/runs/541671215

Reply:
(And on all the other actions too)

Reply:
Hmmm I'm not sure because just 'botify' is used [here](https://github.com/Expensify/Expensify.cash/blob/c15d8f1275a6e0c363bce1036d39fe4c6feae065/.github/workflows/version.yml#L30) and [here](https://github.com/Expensify/Expensify.cash/blob/f0e0c87735bb02f171611bf26055abfe93c0fa17/.github/workflows/cla.yml#L36)

Reply:
Actually I think you're right, changed it to 'OSBotify'